app/views/users/my_sessions.py

```diff
@@ -61,6 +61,8 @@ def display_session_view(session_id):
     return render_template('gentelella/users/mysessions/mysession_detail.html',
                            session=session,
                            speaker_form=speaker_form,
+                           end_time=session.end_time,
```

Comment:
Why do you need to pass these two unnecessarily ? Endtime can be accessed with `session.end_time` in the template and the current time can be accessed with `datetime_now`